packages/components/src/IconDropdownButton/IconDropdownButton.stories.tsx

```diff
@@ -1,33 +1,213 @@
-      <IconDropdownButton {...args}>
-        <MenuItem onClick={() => console.log('v1')}>Вариант выбора 1</MenuItem>
-        <MenuItem onClick={() => console.log('v2')}>Вариант выбора 2</MenuItem>
-        <MenuItem onClick={() => console.log('v3')}>Вариант выбора 3</MenuItem>
-      </IconDropdownButton>
-    </Stack>
-  </Stack>
+type DataType = {
+  id: string;
+  name: string;
+  code: string;
+  ref: string;
+};
+
+const slicedData: DataType[] = [
+  {
+    id: '1',
+    name: 'Офис на ул. Куйбышева 128, к. №1',
+    code: '456-875',
+    ref: 'http://referal.lk',
+  },
+  {
+    id: '2',
+    name: 'Офис на Огарева',
+    code: '456-875',
+    ref: 'http://referal.lk',
+  },
+  {
+    id: '3',
+    name: 'Офис на Маяковской, оф 245',
+    code: '456-875',
+    ref: 'http://referal.lk',
+  },
+];
+
+const handleClick = () => console.log('clicked');
+
+const children = [
+  <MenuItem key={1} onClick={() => console.log('v1')}>
+    Вариант выбора 1
+  </MenuItem>,
+  <MenuItem key={2} onClick={() => console.log('v2')}>
+    Вариант выбора 2
+  </MenuItem>,
+  <MenuItem key={3} onClick={() => console.log('v3')}>
+    Вариант выбора 3
+  </MenuItem>,
+];
+
+const columns: DataGridColumns<DataType>[] = [
+  {
+    field: 'name',
+    label: 'Наименование',
+  },
+  {
+    field: 'code',
+    label: 'Код партнёра',
+  },
+  {
+    field: 'ref',
+    label: 'Реф. ссылка',
+  },
+  {
+    label: '',
+    align: 'center',
+    width: '1%',
+    renderCell: () => (
+      <Stack gap={1} direction="row" justifyContent="center">
+        <IconButton variant="light" onClick={handleClick}>
+          <EditOutlineMd />
+        </IconButton>
+        <IconDropdownButton variant="light" icon={<DotsVOutlineMd />}>
+          {children}
+        </IconDropdownButton>
+      </Stack>
+    ),
+  },
+];
+
+export const IconDropdownButtonShowcase: Story = () => {
+  return (
+    <ExampleTemplate>
+      <Typography variant="h3" paragraph>
+        IconDropdownButton
+      </Typography>
+      <br />
+      <br />
+
+      <Typography variant="ui" paragraph>
+        IconDropdownButton - позволяет компактно разместить несколько вариантов
+        действия в одном элементе.
+      </Typography>
+      <Typography variant="ui" paragraph>
+        В большинстве случаев в качестве такого элемента выступает кнопка или
+        пункт меню. Вызывается по клику или наведению на элемент.
+      </Typography>
+      <br />
+
+      <ExampleTemplate.Case
+        title="Light"
+        descriptionList={[
+          'Кнопка может помещаться на одной строке с другими компонентами и не предполагает привлечения обязательного внимания всех пользователей. При взаимодействии со страницей часть пользователей не воспользуется кнопкой, но она может быть нужна определенной группе пользователей.',
+        ]}
+      >
+        <Stack gap={8} direction="row" justifyContent="center">
+          <IconDropdownButton variant="light" icon={<DotsVOutlineMd />}>
```

Comment:
возможно стоит добавить aria-label='Меню' во все IconDropdownButton, чтобы не забывали о доступности
```suggestion
          <IconDropdownButton variant="light" icon={<DotsVOutlineMd />} aria-label='Меню'>
```